packages/client/hmi-client/src/components/drilldown/tera-drilldown-header.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+	<header>
+		<div class="title-row">
+			<div class="title-container">
```

Comment:
Feel free to use [CSS Grid](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Grids) instead of having `<div>` everywhere that do nothing but just convenient styling.

Reply:
My go to for layouts has usually been to use Flexbox hence the divs.   I can look into using the Grid styling more

Reply:
for now I chagned this up a bit so that the title-container isn't even needed.
```
<div class="title-row">
  <h4><slot /> <i v-if="props.tooltip" v-tooltip="tooltip" class="pi pi-info-circle" /></h4>
  
  <Button
  class="close-mask"
  icon="pi pi-times"
  text
  rounded
  aria-label="Close"
  @click="emit('close')"
  />
  </div>
```